pages/hpc/jobs.md

```diff
@@ -148,6 +148,10 @@ echo "Hello world from faculty cluster node: "`/usr/bin/uname -n`
 #Let's finish some work
 
 ```
+!!! Note 
+	Jobs on the faculty cluster are allowed to run up until the downtime starts. Please ensure your jobs checkpoint and can restart where they left off OR request only enough time to run your job prior to the 7am cutoff.
+
```

Comment:
We know this refers to the maintenance downtimes but it's not clear when reading from top to bottom.  Recommend something like:
```suggestion
!!! Note "Caution: Maintenance Downtimes"
    Jobs on the faculty cluster are allowed to run up until the downtime starts. Please ensure your jobs checkpoint and can restart where they left off OR request only enough time to run your job prior to the 7am cutoff on maintenance days.  See the [schedule here](https://ubccr.freshdesk.com/support/discussions/forums/5000296650)
```